### PR TITLE
Adding minimum and maximum amount for items

### PR DESCRIPTION
**Pull request by heinenandreas**
Adding minimum and maximum amount for items.

Inital styling for the itemlist.

Created a Itemlist component to keep the code in index.js clear

### CODE REVIEW

File: components/GlobalStyle/GlobalStyle.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+      box-sizing: border-box;
+      --darkblue: #2d2861;
+      --orange: #fe7902;
+      --lightyellow: #fcc13f;
+      --green: #fe7902;
+      --lightgreen: #8ad234;
```
Comment (JelenadlS) [inline components/GlobalStyle/GlobalStyle.js:10]: I guess this should be in root? 😊

File: components/GlobalStyle/GlobalStyle.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      --pink: #d6007e;
+      --neonpink: #fc008b;
+      margin: 0;
+}
+
+:root {
```
Comment (NikRoe) [inline components/GlobalStyle/GlobalStyle.js:16]: your root is empty. if something is empty, you can leave it out

File: components/Itemlist/Itemlist.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+/* Style for amount-Headline */
+const StyledAmountHeadline = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding-right: 1em;
+  gap: 1.5em;
+  align-items: flex-end;
+
```
Comment (JelenadlS) [inline components/Itemlist/Itemlist.js:36]: here is one empty row too much 😊

**Review by JelenadlS — COMMENTED** [review]
Hey Andreas, as we are working differently and i do not know the instructions you got, i will just leave you some comments 😊
However, I think your lists looks good so far 😊 Just also checked vercel and its not really responsive with the border around 😉

File: components/GlobalStyle/GlobalStyle.js (hunk 1 above)
Comment (heinenandreas) [reply components/GlobalStyle/GlobalStyle.js:10]: Thank you for your suggestions Jelena :)

**Pull request merged**